content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,257 @@
-appear in each column). An option to display the columns in ascending or descending order appears at
-the end of the options.
+   :alt: Menu buttons with the bar chart called out, along with the other option buttons.
 
 .. tip::
-   Clicking an option enables it. To disable the option, click it again. When the option is enabled
-   it appears grey. When it is inactive, it appears white.
+   Clicking an option enables it. To turn off the option, click it again. When the option is enabled
+   it appears gray. When it is inactive, it appears white.
 
 Pie chart
 ~~~~~~~~~
 
-To display the data in a pie chart, click on the :guilabel:`Pie Chart` icon (last icon) in the menu
-bar. There are no additional options available in this view.
+To display the data in a pie chart, click on the :guilabel:`Pie Chart` icon in the menu bar. There
+are no additional options available in this view.
 
 .. image:: reporting/pie-chart.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Menu buttons with the pie chart called out.
 
 Pivot table
 ~~~~~~~~~~~
 
-The bottom half of the :guilabel:`Payroll Analysis` dashboard presents a pivot table no matter which
-type of chart is selected for the top of the dashboard. The default information displayed is the
-number of payslips, net wage, gross wage, days of paid time off, and days of unpaid time off. The
-information is divided by department. To display more information on the report, select the
-:guilabel:`Measures` drop-down menu, then click on any other items to display.
+To display the data in a pivot table, click on the :guilabel:`Pivot` icon located in the far-right
+side of the top menu bar.
 
-.. image:: reporting/measures-bottom.png
+The default information displayed includes the number of payslips (:guilabel:`# Payslip`), the
+:guilabel:`Net Wage`, the :guilabel:`Gross Wage`, the number of :guilabel:`Days of Paid Time Off`,
+and the number of :guilabel:`Days of Unpaid Time Off`. The information is organized by department.
+
+To display more information on the report, click the :guilabel:`Measures` button to reveal a
+drop-down menu. Then, click on any other metric to display it.
+
+.. image:: reporting/pivot.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Pivot table view with the various metrics called out.
 
 To sort the entries by a specific column, such as :guilabel:`Net Wage`, click on the column name
 twice. The first click selects the column, and the second click sorts the information in descending
 order.
 
-.. image:: reporting/net-wage.png
-   :align: center
-   :alt: To sort information by descending order, click the column head twice.
-
-To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information will
-be downloaded into an Excel spreadsheet.
+To export the data in an XLSX format, click the :guilabel:`Download xlsx` icon. The information is
+then downloaded into a spreadsheet.
```

Comment:
A little clarity to help the reader locate this icon may be useful -- perhaps even a more rudimentary description could be beneficial

Reply:
I agree- that's why I had added the image after it, since it's an odd-looking icon. I like adding the description- thank you.  I tried my best to describe it correctly- feel free to change it though!